native/build.gradle

```diff
@@ -16,4 +16,5 @@ repositories {
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compile 'com.github.jansorg:BashSupport:idea-171.x-SNAPSHOT'
-}
\ No newline at end of file
+    compile fileTree(dir: '/home/alberto/opt/idea-IC-171.3780.107/lib', include: ['*.jar'])
```

Comment:
You can just put here something like `vendor/idea/lib` and download+extract it in the Makefile if it does not exist.